ansible/roles/amazon-cloudwatch-agent/handlers/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: restart collectd
+  ansible.builtin.service:
+    name: collectd
+    state: restarted
+    enabled: yes
```

Comment:
If the purpose of the handler is to restart collectd, I would just do that and not enable it as well.  (Although to be fair, other roles do the same)